src/dotnet-interactive-vscode-insiders/grammars/http.language-configuration.json

```diff
@@ -0,0 +1,5 @@
+{
+    "comments": {
+        "lineComment": "//"
+    }
+}
```

Comment:
Arguably there should be brackets support here as well because of our support for interpolated variables.

Reply:
fixed